include/net/net_if.h

```diff
@@ -469,6 +469,10 @@ struct net_if {
 
 	/** Network interface instance configuration */
 	struct net_if_config config;
+
+	/** Keep track of packets pending in traffic queues */
+	int tx_pending[NET_TC_TX_COUNT];
+	int rx_pending[NET_TC_RX_COUNT];
```

Comment:
Do we need to track these? Currently, net pm only requires to know about tx queue.